src/index.html

```diff
@@ -15,40 +15,18 @@ <h1>Leaderboard</h1>
       <div class="recent-score">
         <div class="title">
           <h1>Recent scores</h1>
-          <button>Refresh</button>
+          <p>This will show top 10 scores</p>
+          <button class="refresh">Refresh</button>
         </div>
         <div class="scores">
-          <table>
-            <tr>
-              <td>Name</td>
-              <td>100</td>
-            </tr>
-            <tr>
-              <td>Name</td>
-              <td>100</td>
-            </tr>
-            <tr>
-              <td>Name</td>
-              <td>100</td>
-            </tr>
-            <tr>
-              <td>Name</td>
-              <td>100</td>
-            </tr>
-            <tr>
-              <td>Name</td>
-              <td>100</td>
-            </tr>
-            <tr>
-              <td>Name</td>
-              <td>100</td>
-            </tr>
+          <table class="datas">
+
           </table>
         </div>
       </div>
       <div class="add-score">
         <h1>Add your score</h1>
-        <form action="">
+        <form action="" class="formInput">
           <div class="inputs">
             <input type="text" name="name" id="name" placeholder="Your name">
           </div>
```

Comment:
[Optional] It would be nice if you could add some form validations so that users cannot submit empty fields or post text in the score input field